docs/developer-docs/smart-contracts/advanced-features/management-canister.mdx

```diff
@@ -91,33 +91,21 @@ import {
     CreateCanisterResult,
 } from 'azle/canisters/management';
 
-type State = {
-    createdCanisterId: Principal;
-};
-
-let state: State = {
-    createdCanisterId: Principal.fromText('aaaaa-aa')
-};
-
 export default class {
```

Comment:
The syntax here is incorrect, creating a simple Azle project and pasting these in to make sure there are no type errors would help.

```typescript
import { call, update } from 'azle';
import {
    CreateCanisterArgs,
    CreateCanisterResult
} from 'azle/canisters/management';

export default class {
    @update([], CreateCanisterResult)
    async executeCreateCanister(): Promise<CreateCanisterResult> {
        return await call('aaaaa-aa', 'create_canister', {
            paramIdlTypes: [CreateCanisterArgs],
            returnIdlType: CreateCanisterResult,
            args: [{ settings: [], sender_canister_version: [] }],
            payment: 50_000_000_000_000n
        });
    }
}
```

Reply:
Revised example